src/About.js

```diff
@@ -1,10 +1,61 @@
 import React from 'react'
+import './About.css'
+import AlexP from './images/AlexP.jpg'
+import AlexR from './images/AlexR.jpg'
+import AliciaW from './images/AliciaW.jpg'
+import DanielN from './images/DanielN.jpg'
+import JoeS from './images/JoeS.JPG'
+import NickA from './images/NickA.jpg'
+import JacobL from './images/JacobL.jpg'
+import JoshM from './images/JoshM.jpg'
 
 const About = () => {
     return (
         <div>
-            <h2>About:</h2>
-            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+            <h2>About our project</h2>
+            <p className='content'>We all know how your dog can become such an important part of your family. So we all set out to create a convient dog training app with a variety of commands in one spot. Meet the friendly team and the doggos that inspired the project.</p>
+            <div className='dog-container'>
+                <div className='dev'>
+                    <h3>Alex R</h3>
+                    <p>Meet their pups Lugnut and Skeeter</p>
+                    <img src={AlexR} alt='Photo of Alex Rs dogs'/>
+                </div>
+                <div className='dev'>
+                    <h3>Alex P</h3>
+                    <p>Meet their pup Olive</p>
+                    <img src={AlexP} alt='Photo of Alex Ps dog'/>
+                </div>
+                <div className='dev'>
+                    <h3>Alicia W</h3>
+                    <p>Meet their pup Staniel</p>
+                    <img src={AliciaW} alt='Photo of Alicias dog'/>
+                </div>
+                <div className='dev'>
+                    <h3>Joe S</h3>
+                    <p>Meet their pup Bhodi</p>
+                    <img src={JoeS} alt='Photo of Joes dog'/>
+                </div>
+                <div className='dev'>
+                    <h3>Jacob L</h3>
+                    <p>Meet their pup Jojo</p>
+                    <img src={JacobL} alt='Photo of Jacobs dog'/>
+                </div>
+                <div className='dev'>
+                    <h3>Josh M</h3>
+                    <p>Meet their pup Pheonix</p>
```

Comment:
Please change my lil guys name to Phoenix.

Reply:
Fixed